src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs

```diff
@@ -74,10 +75,15 @@ overrideHaskellPackages oc packages = vcat
     , "};"
     , ""
     ]
-  , "in callPackage <nixpkgs/pkgs/development/haskell-modules> {"
+  , "in callPackage (nixpkgsPath + \"/pkgs/development/haskell-modules\") {"
```

Comment:
Why concatenate two constants? The previous version with single path literal looks more clear to me.